src/scenes/Question/scenes/Read/components/Flag/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { capitalize } from 'lodash'
+
+import Chip from 'react-toolbox/lib/chip/Chip'
+import Avatar from 'react-toolbox/lib/avatar/Avatar'
+import Tooltip from 'react-toolbox/lib/tooltip/Tooltip'
+
+const TooltipChip = Tooltip()(Chip)
+
+class Flag extends Component {
```

Comment:
Why don't you use functional component here ?

Reply:
Good tips, will do that 👍